core/client/fs/src/main/java/alluxio/client/RetryHandlingMetaMasterClient.java

```diff
@@ -71,10 +72,13 @@ protected void afterConnect() {
   }
 
   @Override
-  public BackupResponse backup(String targetDirectory,
-                                            boolean localFileSystem) throws IOException {
-    return retryRPC(() -> BackupResponse.fromPoto(mClient.backup(BackupPOptions.newBuilder()
-        .setTargetDirectory(targetDirectory).setLocalFileSystem(localFileSystem).build())));
+  public BackupResponse backup(String targetDirectory, boolean localFileSystem) throws IOException {
```

Comment:
Unrelated to this PR - our user facing client signatures usually have `*POptions` in the signature, do you think we should do the same for the MetaMaster client?

Reply:
Created https://github.com/Alluxio/alluxio/pull/8598